demo/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "tsickle-demo",
+  "version": "0.0.0",
+  "description": "sample app demonstrating tsickle",
+  "main": "demo.js",
+  "license": "Apache-2.0",
+  "private": true,
+  "scripts": {
+    "build": "node_modules/typescript/bin/tsc",
+    "demo": "node build/demo.js -- --outDir `pwd`/tsickle-out"
+  },
+  "dependencies": {
+    "minimist": "^1.2.0",
+    "mkdirp": "^0.5.1",
+    "tsickle": "../bazel-bin/npm_package",
```

Comment:
xxx should this be actual `tsickle`? you can still npm link it for local dev, I think.

Reply:
Done.